KsApi/mutations/UpdateUserProfileMutation .swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+public struct UpdateUserProfileMutation<T: GraphMutationInput>: GraphMutation {
+  var input: T
+
+  public init(input: T) {
+    self.input = input
+  }
+
+  public var description: String {
+    return """
+    mutation updateUserCurrency($input: UpdateUserProfileInput!) {
```

Comment:
Let's rename this mutation to `updateUserProfile` for consistency with the name of the swift `struct`